package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@mui/material": "^5.3.0",
```

Comment:
@psahithireddy @harshsharma6401 opinions on this?

Reply:
It is not being used at this point. But there are chances that it might come into use. If we look at the current situation we should merge it but if we are looking in the future I can add that also!

Reply:
I have added @mui/icons-material as a dependency.